helm_deploy/values-dev.yaml

```diff
@@ -25,6 +25,7 @@ generic-service:
     AUDIT_ENABLED: 'true'
     FEATURE_RELATIONSHIP_APPROVED_BY_ENABLED: 'true'
     FEATURE_SEARCH_BY_CONTACT_ID_ENABLED: 'true'
+    FEATURE_NOMIS_CONTACT_OFF_PILOT_ENABLED_PRISONS: 'KMI,GNI,SPI,LGI,DWI,HOI,WWI,LEI,LWI,BWI,BLI,SLI,IWI,LCI,LFI,LHI'
```

Comment:
Not 100% sure of the meaning with the name.

A bit of OCD for me here but can the prison codes be in alphabetical order?

Easier to quickly scan the prison codes that way IMO, particularly if there are a lot of them.